docs/concepts/work-pools.md

```diff
@@ -0,0 +1,273 @@
+
+For example:
+<div class="terminal">
+
+```bash
+$ prefect agent start --match "foo-"
+```
+
+</div>
+
+This example will poll every work queue that starts with "foo-".
+
+### Configuring prefetch
+
+By default, the agent begins submission of flow runs a short time (10 seconds) before they are scheduled to run. This allows time for the infrastructure to be created, so the flow run can start on time. In some cases, infrastructure will take longer than this to actually start the flow run. In these cases, the prefetch can be increased using the `--prefetch-seconds` option or the `PREFECT_AGENT_PREFETCH_SECONDS` setting. Submission can begin an arbitrary amount of time before the flow run is scheduled to start. If this value is _larger_ than the amount of time it takes for the infrastructure to start, the flow run will _wait_ until its scheduled start time. This allows flow runs to start exactly on time.
+
+## Work pool overview
+
+Work pools organize work that [agents](#agent-overview) pick up for execution. Deployments and agents coordinate through a shared work pool name. 
+
+!!! tip "Work pools are like pub/sub topics"
+    It's helpful to think of work pools as a way to coordinate (potentially many) deployments with (potentially many) agents through a known channel: the pool itself. This is similar to how "topics" are used to connect producers and consumers in a pub/sub or message-based system. By switching a deployment's work pool, users can quickly change the agent that will execute their runs, making it easy to promote runs through environments or even debug locally.
+
+In addition, users can control aspects of work pool behavior like how many runs the pool allows to be run concurrently or pausing delivery entirely. These options can be modified at any time, and any agent processes requesting work for a specific pool will only see matching flow runs.
+
+### Work pool configuration
+
+!!! tip "Advanced topic"
+    Work pools do not require manual creation or configuration, because Prefect will automatically create them whenever needed. Managing work pools offers advanced control over how runs are executed.
+
+You can configure work pools by using:
+
+- Prefect UI [**Work pools**](/ui/work-pools/) page
+- Prefect CLI commands
+- Prefect Python API
+
+![Creating a new work pool in the Orion UI](/img/ui/work-pool-create.png)
+
+
+To configure a work pool via the Prefect CLI, use the `prefect work-pool create` command:
+
+<div class="terminal">
+```bash
+prefect work-pool create [OPTIONS] NAME
+```
+</div>
+
+`NAME` is a required, unique name for the work pool.
+
+Optional configuration parameters you can specify to filter work on the pool include:
+
+| Option      | Description                                                         |
+| ----------- | ------------------------------------------------------------------- |
+| -t, --type  | The infrastructure type of the work pool.                           |
```

Comment:
we should avoid exposing this right now, it should have a default

Reply:
I've removed it from this page. It was in the CLI code. I'll check to confirm that its hidden.